packages/api-explorer/e2e/diffScene.spec.ts

```diff
@@ -44,8 +44,9 @@ describe('Diff Scene', () => {
     await jestPuppeteer.resetBrowser()
     await page.setDefaultNavigationTimeout(120000)
   })
-  it('loads the default scene (/diff/3.1)', async () => {
-    await goToPage(`${BASE_URL}/diff/3.1`)
+
+  it('loads the default scene (/3.1/diff)', async () => {
```

Comment:
reminds me we need to change this default .. it should be 4.0 already based on spec status

Reply:
agreed